src/screens/HomeScreen.js

```
@@ -628,22 +1007,55 @@ M7muBbF0XN7VO80iJPv+PmIZdEIAkpwKfi201YB+BafCIuGxIF50Vg==
+                        padding: 8,
+                      }}>
+                      <View style={{}}>
+                        <MaterialIcons
+                          name="workspace-premium"
+                          size={24}
+                          color={COLORS.red}
+                        />
+                      </View>
+                    </TouchableOpacity>
+                  )}
+                </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)